test/functional/aggregation.test.js

```diff
@@ -938,12 +933,9 @@ describe('Aggregation', function () {
 
     test: function (done) {
       var databaseName = this.configuration.db;
-      var client = this.configuration.newClient(this.configuration.writeConcernMax(), {
-        poolSize: 1
-      });
+      var client = this.configuration.newClient({ poolSize: 1 });
 
       const testCases = [
-        { readConcern: { level: 'local' } },
         { writeConcern: { j: true } },
         { readConcern: { level: 'local' }, writeConcern: { j: true } }
       ];
```

Comment:
This test was passing because the write concern was inherited from the client, making it look like using read concern with explain would fail on the server, when really it was the inherited write concern causing the server to return an error.

Reply:
I'm a little confused about this, why do we want to stop throwing an error if you use explain + readConcern? To me it looks like this test would still be passing due to the an error generated by the driver if not for the change [here](https://github.com/mongodb/node-mongodb-native/pull/2622/files#diff-9cf3d2934bb06056f35b09e0e20d67d0da15324fe2090c45a4a54dc417fab74dR68).

Reply:
The server doesn't actually throw an error when you just use explain & readConcern, so it is unnecessary to throw from the driver in this case. If you remove the driver error, the server would return an error for this test case because the write concern is inherited from the client, meaning explain is being used with writeConcern, not because there is any issue with explain & readConcern.  Does that help?

Reply:
To clarify what I'm confused about: this test was expecting the failure to happen client-side rather than the server-side, and from what I can see it was only the change to `AggregateOperation` that caused this client-side error to stop happening, so I'm not sure I see how the server throwing an error because of the inherited write concern would effect this test. I'd expect a server-side error to cause the test to fail with the `Expected aggregation to fail on client instead of server for option` message.

I think we were using this error as a hint to the user that read concern isn't supported by the explain operation, rather than merely as a way of avoiding a server-side error, and that we're no longer providing that guidance to the user with this change. I can see an argument in favor of this - they just want to get their explain results, and we should just ignore the read concern the way the server does; just wanted to fully understand the reason for the change.

Reply:
Ah, thanks for the clarification, I was misunderstanding.

I agree with what you're saying at the end. I think it makes sense to not error, since the server also ignores the read concern and since it's not really an error condition, in my opinion, in that nothing has gone wrong/is in an invalid state. I could imagine including a warning or possibly something in the documentation (for all of the explainable operations, not just aggregate), if we feel like the user needs this guidance. What do you think?

Reply:
I agree it seems unnecessary to warn about a read concern on explain operations. Is there any additional context you could provide about this @mbroadst ?

Reply:
Unfortunately, I can't recall the rationale for that check, I think it was added before my time and we don't have any specifications covering this. We have a similar situation [here](https://github.com/mongodb/node-mongodb-native/blob/3.6/lib/core/wireprotocol/query.js#L199-L203) where we throw a client-side error if a non-local `readConcern` is provided for servers which don't support it - the server would otherwise ignore the value, it's not clear why we throw an error client-side.

I'm pro this change if it doesn't break any of our tests